csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll

```diff
@@ -25,6 +26,19 @@ class SystemTextStringBuilderClass extends SystemTextClass {
   Method getAppendFormatMethod() { result = this.getAMethod("AppendFormat") }
 }
 
+/** Clear content for `System.Text.StringBuilder.Clear`. */
+private class SystemTextStringBuilderClearFlow extends SummarizedCallable {
+  SystemTextStringBuilderClearFlow() {
+    this.getDeclaringType() instanceof SystemTextStringBuilderClass and
+    this.hasName("Clear")
```

Comment:
Alternatively
```ql
this = any(SystemTextStringBuilderClass c).getAMethod("Clear")
```

Reply:
Yes - great - that looks much nicer! Thank you!